stan/math/opencl/constants.hpp

```diff
@@ -3,8 +3,34 @@
 #ifdef STAN_OPENCL
 namespace stan {
 namespace math {
-enum class TriangularViewCL { Lower = 0, Upper = 1, Entire = 2 };
-enum class TriangularMapCL { UpperToLower = 0, LowerToUpper = 1 };
+enum class triangular_view_CL { LOWER = 0, UPPER = 1, ENTIRE = 2 };
+enum class triangular_map_CL { UPPER_TO_LOWER = 0, LOWER_TO_UPPER = 1 };
+
+namespace opencl_kernels {
+/**
+ * Returns the enum string for OpenCL kernels
+ *
+ * @return enum string
+ */
+inline std::string get_opencl_enum_string() {
```

Comment:
I am not really sure this the best solution for this. I am open to any better ideas here.